docs/virtualenv.rst

```diff
@@ -0,0 +1,45 @@
+.. _virtualenv-heading:
+
+Virtualenv
+=========================
+
+Why use virtualenv?
```

Comment:
These headings should be changed to Title Case for consistency with the rest of the Click docs.